{"error": null, "issues": [{"address": 695, "code": ";\n \n/**\n * @title ", "debug": "storage_16: 0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff\nstorage_keccac_1461501637330902918203684832716283019655932542975_&\n1461501637330902918203684832716283019655932542975_&\ncaller: 0x0\ncalldatasize_TenderToken: 0x3\n", "description": "A possible integer overflow exists in the function `fallback`.\nThe addition or multiplication may result in a value higher than the maximum representable integer.", "filename": "/unique_chucks/22/0xea38c95b650d2145cf96eab2988ac9d0f0b0e387.sol", "function": "fallback", "lineno": 1, "title": "Integer Overflow ", "type": "Warning"}, {"address": 824, "code": "fer(_from, _to, _value);\n ", "debug": "callvalue: 0x57f25cd000000000000000000000000000000000000000000000000000000000\nstorage_18: 0xa8795cd3fffffffffffffffffffffffffffffffffffffffffffffffffffffffe\nstorage_keccac_1461501637330902918203684832716283019655932542975_&\n1461501637330902918203684832716283019655932542975_&\ncaller: 0x0\ncalldatasize_TenderToken: 0x3\n", "description": "A possible integer overflow exists in the function `fallback`.\nThe addition or multiplication may result in a value higher than the maximum representable integer.", "filename": "/unique_chucks/22/0xea38c95b650d2145cf96eab2988ac9d0f0b0e387.sol", "function": "fallback", "lineno": 376, "title": "Integer Overflow ", "type": "Warning"}, {"address": 4435, "code": "2900; // 2.9%\n    uint256 public buyComiss", "debug": "", "description": "This contract executes a message call to an address provided as a function argument. Generally, it is not recommended to call user-supplied addresses using Solidity's call() construct. Note that attackers might leverage reentrancy attacks to exploit race conditions or manipulate this contract's state.", "filename": "/unique_chucks/22/0xea38c95b650d2145cf96eab2988ac9d0f0b0e387.sol", "function": "withdrawToken(address,address,uint256)", "lineno": 328, "title": "Message call to external contract", "type": "Warning"}, {"address": 4435, "code": "2900; // 2.9%\n    uint256 public buyComiss", "debug": "", "description": "Multiple sends exist in one transaction, try to isolate each external call into its own transaction. As external calls can fail accidentally or deliberately.\nConsecutive calls: \nCall at address: 4670\n", "filename": "/unique_chucks/22/0xea38c95b650d2145cf96eab2988ac9d0f0b0e387.sol", "function": "withdrawToken(address,address,uint256)", "lineno": 328, "title": "Multiple Calls", "type": "Information"}, {"address": 4670, "code": "    uint256 public priceUnits = 1 ether", "debug": "", "description": "This contract executes a message call to an address provided as a function argument. Generally, it is not recommended to call user-supplied addresses using Solidity's call() construct. Note that attackers might leverage reentrancy attacks to exploit race conditions or manipulate this contract's state.", "filename": "/unique_chucks/22/0xea38c95b650d2145cf96eab2988ac9d0f0b0e387.sol", "function": "withdrawToken(address,address,uint256)", "lineno": 332, "title": "Message call to external contract", "type": "Warning"}, {"address": 5426, "code": "l += msg.value;\n        }\n", "debug": "The exception is triggered under the following conditions:\n\ncalldata_TenderToken_4: 0x0\nstorage_16: 0x0\ncalldata_TenderToken_0: 0x22f85eaa00000000000000000000000000000000000000000000000000000000\ncalldatasize_TenderToken: 0x4\ncallvalue: 0x0\n", "description": "A reachable exception (opcode 0xfe) has been detected. This can be caused by type errors, division by zero, out-of-bounds array access, or assert violations. This is acceptable in most situations. Note however that `assert()` should only be used to check invariants. Use `require()` for regular input checking. ", "filename": "/unique_chucks/22/0xea38c95b650d2145cf96eab2988ac9d0f0b0e387.sol", "function": "_function_0x22f85eaa", "lineno": 364, "title": "Exception state", "type": "Informational"}, {"address": 10857, "code": ";\n \n/**\n * @title ", "debug": "storage_15: 0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff\nstorage_keccac_1461501637330902918203684832716283019655932542975_&\n1461501637330902918203684832716283019655932542975_&\ncaller: 0x0\ncalldata_TenderToken_32 + 4: 0x0\ncalldata_TenderToken_32 + 36: 0x0\ncalldata_TenderToken_0: 0x23b872dd00000000000000000000000000000000000000000000000000000000\ncalldatasize_TenderToken: 0x4\ncallvalue: 0x0\n", "description": "A possible integer overflow exists in the function `transferFrom(address,address,uint256)`.\nThe addition or multiplication may result in a value higher than the maximum representable integer.", "filename": "/unique_chucks/22/0xea38c95b650d2145cf96eab2988ac9d0f0b0e387.sol", "function": "transferFrom(address,address,uint256)", "lineno": 1, "title": "Integer Overflow ", "type": "Warning"}, {"address": 10986, "code": "on setBuyComission(uint ", "debug": "storage_17: 0x9c9efb7ae5acd378025256a3945f5859076e809243093d9ecae2636c9fd28979\ncalldata_TenderToken_32 + 36: 0xc290137095b13347e55005a6dc184559e0489f1233480050d0fb4bba17d34179\ncalldata_TenderToken_32 + 4: 0x0\nstorage_keccac_1461501637330902918203684832716283019655932542975_&\n1461501637330902918203684832716283019655932542975_&\ncaller: 0xc290137095b13347e55005a6dc184559e0489f1233480050d0fb4bba17d34179\ncalldata_TenderToken_0: 0x23b872dd00000000000000000000000000000000000000000000000000000000\ncalldatasize_TenderToken: 0x4\ncallvalue: 0x0\n", "description": "A possible integer overflow exists in the function `transferFrom(address,address,uint256)`.\nThe addition or multiplication may result in a value higher than the maximum representable integer.", "filename": "/unique_chucks/22/0xea38c95b650d2145cf96eab2988ac9d0f0b0e387.sol", "function": "transferFrom(address,address,uint256)", "lineno": 388, "title": "Integer Overflow ", "type": "Warning"}, {"address": 11461, "code": " a % ", "debug": "The exception is triggered under the following conditions:\n\ncalldata_TenderToken_0: 0x1a70388f00000000000000000000000000000000000000000000000000000000\ncalldatasize_TenderToken: 0x4\nstorage_12: 0x0\ncalldata_TenderToken_4: 0x0\ncallvalue: 0x0\n", "description": "A reachable exception (opcode 0xfe) has been detected. This can be caused by type errors, division by zero, out-of-bounds array access, or assert violations. This is acceptable in most situations. Note however that `assert()` should only be used to check invariants. Use `require()` for regular input checking. ", "filename": "/unique_chucks/22/0xea38c95b650d2145cf96eab2988ac9d0f0b0e387.sol", "function": "shareToWei(uint256)", "lineno": 39, "title": "Exception state", "type": "Informational"}], "success": true}